src/anjaliS/DivisionByWhile.java

```diff
@@ -0,0 +1,53 @@
+package anjaliS;
+
+public class DivisionByWhile{
+	   void even(int start,int end)
+	   {
+	     System.out.println("Even numbers are ");
+	     while(start<=end)
+	     {
+	      if(start%2==0)
+	      System.out.println(start);
+	      start++;
+	     }
+	  }
+	    void divisibleBy5(int start,int end){
+	     System.out.println("Divisible by 5, numbers are:");
+	     while(start<=end)
+	    {
+	     if(start%5==0)
+	      System.out.println(start);
+	      start++;
+	    } 
+	  }
+	    void divisibleBy3and5(int start,int end){
+	     System.out.println("Divisible by 5 & 3, numbers are:");
+	     while(start<=end)
+	   {
+	    if(start%5==0 && start%3==0)
+	     System.out.println(start);
+	     start++;
+	   }
+	 }
+	   void divisibleBy13or7(int start,int end){
+	    System.out.println("Divisible by 7 or 13, numbers are");
+	    while(start<=end)
+	   {
+	    if(start%7==0 )
+	     System.out.println(start +" is divisible by 7");
+	     start++;
+	    if(start%13==0)
+	     System.out.println(start +" is divisible by 13");
+	     start++;
```

Comment:
Incorrect code, increment always happens once only. did you compare your output with expected output? @AnjaliSoct2020

Reply:
yes checked but i think got missed, corrected now ,shall i push again code correct one??

Reply:
yes checked but i think got missed, corrected now ,shall i push again code correct one??@harshvegada